pages/checkout/redirect.tsx

```diff
@@ -113,29 +109,31 @@ export const getServerSideProps: GetServerSideProps = async ({ req, query }) =>
 
   const token = await getToken({ req })
 
-  if (typeof token?.accessToken !== "string") {
-    return { props: defaultPageProps }
+  if (!token || !token.accessToken || typeof token.accessToken !== "string") {
```

Comment:
this change seems unnecessary 🤔

Reply:
Maybe? It was mainly done to align with the rest of the code base 😅  It was done like that in /assets/{id}, then I updated it in /terms and now here in /redirect. Should either keep it, or adjust it in the other files? I just want to do this in the same way 👍

Reply:
i mean it all does the same thing, so it really doesn't matter beyond semantics. besides, comparing booleans is cheap. however, my two cents is that this is overly verbose, and `typeof undefined?.key === "type"` will evaluate to false

Reply:
I think we should stick to one of them to avoid confusion at least, since the code is quite similar for the first part across the 3 pages. We can change all 3 to the "short" version?

Reply:
i'm partial to optional chaining, but it's your call